### PR TITLE
Euler totient function

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Article on Euler Totient Function added

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#250

### CODE REVIEW

**Comment by tk315** [comment]
Please review sir @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786 @nidh-ish

File: docs/mathematics/euler-totient-function.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+Iteration 5:
+i = 5
+if (gcd(5,num) == 1)cnt++
+cnt = 4
+```
+
+## Algorithm
+
+- Declare a variable `cnt` and initialize it to `0`.
+
```
Comment (vybhav72954) [inline docs/mathematics/euler-totient-function.mdx:60]: Remove Extra whitelines between bullet points

File: docs/mathematics/euler-totient-function.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+#### Output :
+
+```
+Count of Numbers from 1 to n which are coprime to n is: 12
+```
+
+## Complexity Analysis
+
+- Time Complexity: `O(n)`
+
```
Comment (vybhav72954) [inline docs/mathematics/euler-totient-function.mdx:158]: Remove the extra whiteline

File: docs/mathematics/euler-totient-function.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+- Space Complexity: `O(1)`
```
Comment (vybhav72954) [inline docs/mathematics/euler-totient-function.mdx:159]: Missing Credits?

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: docs/mathematics/euler-totient-function.mdx (hunk 3 above)
Comment (tk315) [reply docs/mathematics/euler-totient-function.mdx:159]: Sir I have written the code for both c++ and python

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**